playbooks/libvirt/openshift-cluster/config.yml

```diff
@@ -14,10 +14,10 @@
 
 - include: ../../common/openshift-cluster/config.yml
   vars:
-    g_etcd_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-etcd' }}"
-    g_lb_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-lb' }}"
-    g_masters_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-master' }}"
-    g_nodes_group: "{{ 'tag_env-host-type-' ~ cluster_id ~ '-openshift-node' }}"
+    g_etcd_group: "{{ groups['tag_host-type-openshift-etcd'] | intersect(groups['tag_environment_' ~ cluster_id]) }}"
```

Comment:
Are you sure this has tag_environment_ ?

Like GCE, I suspect libvirt uses dashes, and it probably didn't get changed from env to environment.